### ISSUE TITLE
[UX] Views dialogs are waaaaaaaay to wide

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#2696]
## Describe your issue or idea

In previous versions of the Views UI, most dialogs were about the right size. Occasionally, there was one that felt a bit cramped, but most of the time the user experience was okay.

In more recent versions of Backdrop (1.5+?), all of the dialogs I have opened seem to be the same size - waaaaaaaay to large. In this screenshot, you'll see a dialog with a simple yes/no question that spans almost the entire width of my screen!?!

<img width="1628" alt="screen shot 2017-05-15 at 5 00 55 pm" src="https://cloud.githubusercontent.com/assets/397895/26084333/2d8caab6-3990-11e7-9885-2896f9c15301.png">

Not only is this use of space unnecessary, but it creates a user experience regression. I now need to do a lot of thinking and scanning just to find out where my attention should be - in this case I need to look way off to the left, even farther left *than anything else on my site*.

### Steps to reproduce (if reporting a bug)

* Create or edit a view
* Click on any link to edit a setting

### Actual behavior (if reporting a bug)

* Dialog uses all available space whether needed or not

### Expected behavior (if reporting a bug)

* Dialog uses appropriate space for the task

### Relevant version/system information (if applicable)

Possibly related issues:
https://github.com/backdrop/backdrop-issues/issues/741

**Comment by jenlampton** [comment]
Looking over https://github.com/backdrop/backdrop-issues/issues/741 again, it would seem that we've solved the use-case for a task that needs a larger modal size, by making all the modals large. This "breaks" the UX for all the ones that were the right size to start with. Ideally, each task would be able to specify the size it needs, with a sensible middle-of-the-road default that helps the user focus on the task at hand, instead of putting it "off the screen" by default.

**Comment by klonos** [comment]
Yeah, another UX- here is that the user needs to move their mouse greater distances in order to interact with the form elements (radio buttons in this case) and then hit the submit button.

The ideal would be to have a way to dynamically calculate an optimal height vs width ratio or something taking the available screen size into consideration of course. Seems tricky though because too wide has the implications mentioned here; too narrow will cause the dialog to have scrollbars, which is not the best UX either.

**Comment by klonos** [comment]
...I wish I could help with this, but do not see any way to achieve this with pure css (and my JS skills are laughable 😅).
